apigo/api/preview.go

```diff
@@ -103,17 +145,30 @@ func Preview(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 	}
 	log.Infof("Preview request: %+v", body)
 
+	authorizationHeader := r.Header.Get("Authorization")
+	idToken := r.Header.Get("IdToken")
```

Comment:
as discussed, get rid of IdToken in river (always use /user-info in pyrog-server)